### PR TITLE
Release notes for 4.8.1

### PR DESCRIPTION
**Pull request by lorenrh**
Signed-off-by: Lorena Rangel <lorena.rangel@docker.com>

### CODE REVIEW

**Comment by aevesdocker** [comment]
Hi @lorenrh, could you also add the rest of the release notes to 4.8.1, so not just bug fixes, but also the upgrades/new features etc.? 
It would also be great if we could add a section 'Known issues' to the bottom 4.8.0 notes that detail the Kubernetes issue. Something like 'Currently, if you are running a Kubernetes cluster, it will be deleted when you upgrade to Docker Desktop 4.8.0. We aim to fix this in the next release.'

File: desktop/mac/release-notes/index.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ This page contains information about the new features, improvements, known issue
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
 ## Docker Desktop 4.8.0
```
Comment (aevesdocker) [inline desktop/mac/release-notes/index.md:26]:
```suggestion
## Docker Desktop 4.8.1
```

File: desktop/mac/release-notes/index.md [hunk 2]
```diff
@@ -48,6 +48,50 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
```
Comment (usha-mandya) [inline desktop/mac/release-notes/index.md:48]: @lorenrh Can we add info on New, Upgrades,etc?

**Comment by aiordache** [comment]
@lorenrh Please update the Linux release notes too :sweat_smile:

**Comment by lorenrh** [comment]
@aiordache  Not sure if linux users are affected by this 🤔  But I've added the relnotes anyway.

File: desktop/linux/release-notes/index.md [hunk 3]
```diff
@@ -10,6 +10,23 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
+## Docker Desktop 4.8.0
```
Comment (aevesdocker) [inline desktop/linux/release-notes/index.md:13]:
```suggestion
## Docker Desktop 4.8.1
```

**Review by aevesdocker — APPROVED** [review]
LGTM, thank you Lorena!

File: desktop/linux/release-notes/index.md [hunk 4]
```diff
@@ -10,6 +10,23 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
+## Docker Desktop 4.8.1
+2022-05-09
+
+> Download Docker Desktop
+>
+> [DEB](https://desktop-stage.docker.com/linux/main/amd64/78933/docker-desktop-4.8.0-amd64.deb) |
```
Comment (StefanScherer) [inline desktop/linux/release-notes/index.md:18]: the three download links need some care to update to the new build number and version number.

Comment (lorenrh) [reply desktop/linux/release-notes/index.md]: Thanks! It's been updated.

File: desktop/linux/release-notes/index.md [hunk 5]
```diff
@@ -10,6 +10,23 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
+## Docker Desktop 4.8.1
+2022-05-09
+
+> Download Docker Desktop
+>
+> [DEB](https://desktop-stage.docker.com/linux/main/amd64/78998/docker-desktop-4.8.0-amd64.deb) |
```
Comment (StefanScherer) [inline desktop/linux/release-notes/index.md:18]: Still needs a tiny correction: 4.8.0 -> 4.8.1 in the file names.

Comment (mat007) [reply desktop/linux/release-notes/index.md:18]: Yes, this is going to be annoying, I think we should align Linux on the Windows and Mac practices, maybe?

**Pull request merged**